### ISSUE TITLE
null is not an object (evaluating 'Object.keys(node)')

### ISSUE THREAD

**Issue by giautm** [jaydenseric/apollo-upload-client#15]
This happen when pass a `null` value for the variable. Add check null at here 

https://github.com/jaydenseric/apollo-upload-client/blob/master/src/helpers.js#L16

**Comment by jaydenseric** [comment]
What can I do specifically to reproduce it? I figured that would not be necessary, as there is a check that each node to recurse into is an object [here](https://github.com/jaydenseric/apollo-upload-client/blob/master/src/helpers.js#L32), and the start node is checked [here](https://github.com/jaydenseric/apollo-upload-client/blob/master/src/helpers.js#L38).

**Comment by giautm** [comment]
Here is my variables:
```
{
    "input": {
        "orderId": "VHJhbnNwb3J0T3JkZXI6NTkyYmUyNDRjZTRmYTkwYmJlYzg0Y2E3",
        "appointment": null,
        "cause": 8,
        "comment": null,
        "signature": [
            {
                "name": "file_0",
                "type": "image/png",
                "uri": "/Users/admin/Library/Developer/CoreSimulator/Devices/C02A708F-7003-4747-BB09-AA35530AFAC8/data/Containers/Data/Application/44243AD1-906B-4C2E-AC28-6D700BA99609/tmp/ReactABI17_0_0Native/CA1AD1AC-2B6D-47EB-9AE3-528A537E8AC7.png"
            }
        ],
        "photos": [],
        "timestamp": "2017-06-20T15:48:34.832Z",
        "clientMutationId": 1497973714832
    }
}
```

**Comment by jaydenseric** [comment]
Thanks; it's 2 am local time so I will have to resolve it tomorrow. In the meantime downgrading to v4.0.4 will help.

**Comment by giautm** [comment]
of course, check `undefined` too. `if (node)` seen be great. 👍

**Comment by jaydenseric** [comment]
Fixed in [v4.0.7](https://github.com/jaydenseric/apollo-upload-client/releases/tag/v4.0.7).
